pkg/debuggers/remote/utils.go

```diff
@@ -53,15 +61,15 @@ func GetPortOfJavaProcess(pid int) (int, error) {
 	for _, arg := range args {
 		port, err = checkAndParseArgument(arg)
 		if err != nil {
-			log.WithFields(log.Fields{"pid": pid, "err": err, "arg": arg}).Error("Can't get command line arguments")
+			log.WithFields(log.Fields{"pid": pid, "err": err, "i": arg}).Error("Can't get command line arguments")
 			break
 		}
 		if port != 0 {
 			break
 		}
 	}
 	if port == 0 {
-		err = fmt.Errorf("Can't find port in java command line arguments for PID: %d, args: %v", pid, args)
+		err = fmt.Errorf("can't find port in java command line arguments for PID: %d, args : %v or env : %v", pid, args, s)
```

Comment:
the meaning of the `s` arg is not clear, could you please use a more descriptive name?

Reply:
shall i call it `envArgs`

Reply:
are we good with conversation, can we mark it resolved? Any todo here?